metrics-operator/cmd/metrics/adapter/provider/custom_metrics.go

```diff
@@ -104,3 +108,11 @@ func generateCustomMetricInfo(name string) provider.CustomMetricInfo {
 		Namespaced: true,
 	}
 }
+
+func getMetricKey(metricName, metricNamespace string) string {
```

Comment:
maybe it is worth adding some extra type safety and introducing `type metricKey string` and having `map[metricKey]CustomMetricValue` so we force to pass through this function all the time